resources/definitions/cubicon_style_neo_a22.def.json

```diff
@@ -1,40 +0,0 @@
-{
-    "version": 2,
```

Comment:
Why are you removing this printer? This will cause configurations to become corrupted.